notes/hashtags.md

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
What's the purpose of this file?

Reply:
I used this file as a template for if you wanted to maybe have a dedicated page with a list of all hashtags. If you don't intend to have a page with a list of all the hashtags then this file could be removed.

Reply:
Removed this file and moved `tags.liquid` into the `/notes/` directory so it renders as an HTML page when building the site instead of being a layout.